scheduler/workflow.go

```diff
@@ -270,11 +280,17 @@ func bindPluginContentTypes(pus []*publishNode, prs []*processNode, mm managesPl
 			}
 		}
 		//continue the walk down the nodes
-		if err := bindPluginContentTypes(pr.PublishNodes, pr.ProcessNodes, mm, rct); err != nil {
+		if err := bindPluginContentTypes(pr.PublishNodes, pr.ProcessNodes, local, rct, clients); err != nil {
 			return err
 		}
 	}
 	for _, pu := range pus {
+		var mm managesPluginContentTypes
+		if pu.Target != "" {
+			mm = clients.Get(pu.Target)
```

Comment:
If clients had a local instance (given at create time) it could return the correct instance without the conditional existing here.